[tts][TCSACR-477] Add guide for Repeat() (#1582)
Samsung/tizen-docs: docs/application/dotnet/guides/uix/tts.md

    ```

<a name="text"></a>

> ## Add Text

Reviewer: I think the second heading capitalization issue was not addressed.
```suggestion
## Add text
```

Author: Thank you for checking.
I also missed it.
I will fix it.

    ```

-   To repeat the last playback, use the `Repeat()` method.

>     The `Repeat()` method returns the information about the text which is last played, and the TTS state is changed to `Playing`.

Author: 네 정확히는 RepeatedText라고 하는 클래스이구요, 해당 클래스에는 마지막으로 재생했던 text string과 해당 text에 대한 utterance id가 포함되어 있습니다. RepeatedText 클래스에 대한 설명도 추가하는것이 더 좋을까요?

    ```

-   To repeat the last playback, use the `Repeat()` method.

    The `Repeat()` method returns the information about the text is played lastly, and the TTS state is changed to `Playing`.

>     If there is no text is played before,  an exception occurs. To handle the exception, contain a block of code as follow: 

Author: As the comment that I leave previously, `Repeat()` uses the lastly added text, so I am going to fix like this.
```suggestion
    If no text has been added before, an exception occurs. To handle the exception, contain a block of code as follow: 
```

    ```csharp
    void Stop()
    {
        try
        {
            string text = "tutorial"; /// Text to be read
            string language = "en_US"; /// Language
            int voice_type = Voice.Female; /// Voice type
            int speed = 0; /// Read speed; 0 for Auto

            tts_inst.AddText(text, language, voice_type, speed);
            tts_inst.Play();

            ...

            tts_inst.Stop();
            RepeatedText text = tts_inst.Repeat();
        }
        catch (Exception e)
        {
            /// Error handling
        }
    }
    ```

> ## Related Information

Author: Thank you for checking.
I will fix it soon.

> * Dependencies

Author: This seems to be error by copying.
I will fix it soon.
Thank you for checking.
